src/gobprepare/importers/dump_importer.py

```diff
@@ -0,0 +1,107 @@
+import contextlib
+from io import StringIO
+import tempfile
+import gzip
+import re
+import tempfile
+from typing import List, Iterator, Optional
+
+from gobprepare.importers.typing import ReadConfig, SqlDumpImporterConfig
+from gobconfig.datastore.config import get_datastore_config
+from gobcore.datastore.factory import DatastoreFactory
+from gobcore.datastore.objectstore import ObjectDatastore
+from gobcore.datastore.sql import SqlDatastore
+from gobcore.logging.logger import logger
+
+from gobcore.exceptions import GOBException
+
+@contextlib.contextmanager
+def _load_from_objectstore(datastore: ObjectDatastore) -> Iterator[str]:
+    datastore.connect()
+
+    obj_info = next(datastore.query(None), None)
+
+    if obj_info is None:
+        raise GOBException(f"File not found on Objectstore: {datastore.read_config['file_filter']}")
+
+    _, obj = datastore.connection.get_object(
+        container=datastore.container_name, obj=obj_info["name"], resp_chunk_size=100_000_000
+    )
+
+    with tempfile.NamedTemporaryFile(mode="wb", suffix=".gz") as fp:
+        try:
+            for chunk in obj:
+                fp.write(chunk)
+            logger.info(f"File loaded from objectstore: {obj_info['name']}")
+            yield fp.name
+        finally:
+            datastore.disconnect()
+
+
+class SqlDumpImporter:
+    """Import a SQL dump into a SqlDatastore table."""
+
+    def __init__(self, dst_datastore: SqlDatastore, config: SqlDumpImporterConfig) -> None:
+        """ Initialise SqlDumpImporter """
```

Comment:
I would expect black/flake8 should complain here. Not enabled yet for this file? Should do that

Reply:
file formatted met black maar levert niks.

Reply:
No I expected it to come back with "comment needs to end with period (.)" or the use of verbs and the space after the first """. Anyhow. If it's formatted with black and flake8, it should be alright :)